### DISCUSSION TITLE
2021-02 Cumulative Update for Windows Server 2016 for x64-based Systems (KB4601318)

### DISCUSSION THREAD

**Discussion by mikkelxyz** [dotnet/aspnetcore#30095] [Q&A]
Hi all

Our API servers were updated today with KB4601318 and that broke our SignalR WebSockets application. To be certain about this, we have tried unstalling the update, which made it work again, and then reinstalling it, which broke it again:
![image](https://user-images.githubusercontent.com/23260688/107644109-4afdff00-6c77-11eb-81db-2958273032ca.png)

Has anyone tried anything similar?

**Comment by BrennanConroy** [comment]
Note: Discussions will be removed some time today. I recommend you file an issue instead https://github.com/dotnet/aspnetcore/issues/29935

Can you share the error you get as text, and also are there any server logs?

And from the small screenshot, it looks like WebSockets wasn't attempted by the client, which can imply you either disabled it on the client, or the server doesn't support it.
